vickev/howdypix
# Fix the UI

## Description

Fix the UI so that is looks the same as https://confluence.vickev.com/pages/viewpage.action?spaceKey=PIX&title=Browse+an+album

![image](https://user-images.githubusercontent.com/219680/68073904-54f23c00-fd6b-11e9-81fe-12a393fb3e3b.png)






## How to test?

- Check-out this branch.
- Run `yarn dev` => Check the UI

## Checklist
